src/main/java/frc/robot/subsystems/ball/feeder/FeederImpl.java

```diff
@@ -12,13 +14,22 @@
     WPI_TalonSRX rollerController;
 
     Ultrasonic feederUltrasonic;
+    UltrasonicPoller poller;
+    SmartDashboardTuner distanceTuner;
+
+    double ballDistance;
 
     public FeederImpl(){
         super(1, TimeUnit.SECONDS);
         
         this.beltController = new WPI_TalonSRX(RobotMap.BELT_TALON_PORT);
         this.rollerController = new WPI_TalonSRX(RobotMap.ROLLER_TALON_PORT);
         this.feederUltrasonic = new Ultrasonic(RobotMap.ULTRASONIC_PORTS[0], RobotMap.ULTRASONIC_PORTS[1]);
+        
+        this.ballDistance = 300;
+        this.distanceTuner = new SmartDashboardTuner("Ball Distance", this.ballDistance, d -> this.ballDistance = d);
+
+        this.poller = new UltrasonicPoller(feederUltrasonic, 3000, this.ballDistance);
```

Comment:
This won't work how you want it to, because the `ballDistance` value is only passed into the UltrasonicPoller constructor once, so any changes you make using the tuner will not actually do anything other than change the field value within `FeederImpl`.

If you put the tuner inside the poller then it'll probably do what you're expecting.